### ISSUE TITLE
twisted.test.test_tcp.callWithSpew is unused

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#7162]
|<img alt="Alex's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| Alex reported|
|-|-|
|Trac ID|trac#7162|
|Type|enhancement|
|Created|2014-04-15 21:37:08Z|
|Branch|https://github.com/twisted/twisted/tree/callWithSpew-7162|

It can therefore be removed.

<details><summary>Searchable metadata</summary>

```
trac-id__7162 7162
type__enhancement enhancement
reporter__Alex Alex
priority__normal normal
milestone__ 
branch__branches_callWithSpew_7162 branches/callWithSpew-7162
branch_author__wsanchez wsanchez
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1397597828000000 1397597828000000
changetime__1474198049187875 1474198049187875
version__None None
owner__wsanchez wsanchez

```
</details>

**Comment by twisted-trac** [comment]
|<img alt="Alex's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_1"></a>|Alex commented|
|-|-|

This is resolved by https://github.com/twisted/twisted/pull/18/files

**Comment by twisted-trac** [comment]
|[<img alt="wsanchez's avatar" src="https://avatars.githubusercontent.com/u/50002?s=50" width="50" height="50">](https://github.com/wsanchez)<a name="note_2"></a>|@wsanchez set owner to @wsanchez<br>@wsanchez set status to `assigned`|
|-|-|

Confirmed unused:

```
[Cab-Calloway:~/Developer/Twisted/trunk] wsanchez$ find . ! \( -type d -name .svn -prune \) -type f -print0 | xargs -0 grep callWithSpew
./twisted/test/test_tcp.py:def callWithSpew(f):
```

**Comment by twisted-trac** [comment]
|[<img alt="wsanchez's avatar" src="https://avatars.githubusercontent.com/u/50002?s=50" width="50" height="50">](https://github.com/wsanchez)<a name="note_3"></a>|@wsanchez commented|
|-|-|

(In [42311]) Branch branch branch

**Comment by twisted-trac** [comment]
|[<img alt="wsanchez's avatar" src="https://avatars.githubusercontent.com/u/50002?s=50" width="50" height="50">](https://github.com/wsanchez)<a name="note_4"></a>|@wsanchez commented|
|-|-|

Tests look good, reviewed.

**Comment by twisted-trac** [comment]
|[<img alt="wsanchez's avatar" src="https://avatars.githubusercontent.com/u/50002?s=50" width="50" height="50">](https://github.com/wsanchez)<a name="note_5"></a>|@wsanchez set status to `closed`|
|-|-|

(In [42318]) Merge branches/callWithSpew-7162: Remove unused twisted.test.test_tcp.callWithSpew.

Author: Alex Gaynor
Reviewer: wsanchez
Fixes: [#7162](https://github.com/twisted/twisted/issues/7162)

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_6"></a>|@hawkowl commented|
|-|-|

[mass edit] Removing review from closed tickets.
